### PR TITLE
Fix textFilter() for Internet Explorer (includes() and find() are not supported) 

### PR DESCRIPTION
**Pull request by NixonK**
Fix textFilter() for Internet Explorer (includes() and find() are not supported) 
- replace includes() with indexOf() !== -1
  - [includes() browser compatibility](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes#Browser_compatibility)
- replace find() with for loop
  - [find() browser compatibility](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find#Browser_compatibility)

### CODE REVIEW

File: packages/react-bootstrap-table2-filter/src/filter.js [hunk 1]
```diff
@@ -106,7 +106,14 @@ export const filters = (store, columns, _) => (currFilters) => {
...
     filterFn = factory(filterObj.filterType);
-    const { filterValue } = columns.find(col => col.dataField === dataField);
+    let filterValue;
+    const keys = Object.keys(columns);
+    for (let i = 0; i < keys.length; i += 1) {
+      if (columns[keys[i]].dataField === dataField) {
+        filterValue = columns[keys[i]].filterValue;
+        break;
+      }
+    }
```
Comment (AllenFang) [inline packages/react-bootstrap-table2-filter/src/filter.js]: I think just use column.length will be much readable:

```js
let filterValue;

for (let i = 0; i < columns.length; i += 1) {
  if (columns[i].dataField === dataField) {
    filterValue = columns[keys[i]].filterValue;
    break;
  }
}
```

File: packages/react-bootstrap-table2/src/sort/wrapper.js [hunk 2]
```diff
@@ -39,8 +39,14 @@ export default Base =>
...
-      const sortedColumn = nextProps.columns.find(
-        column => column.dataField === nextProps.store.sortField);
+      let sortedColumn;
+      const keys = Object.keys(nextProps.columns);
+      for (let i = 0; i < keys.length; i += 1) {
+        if (nextProps.columns[keys[i]].dataField === nextProps.store.sortField) {
+          sortedColumn = nextProps.columns[keys[i]];
+          break;
+        }
+      }
```
Comment (AllenFang) [inline packages/react-bootstrap-table2/src/sort/wrapper.js]: Same as above

**Comment by AllenFang** [comment]
@NixonK thanks your contributions, there's still minor need to patch, thanks!!

**Comment by NixonK** [comment]
@AllenFang Changes made, let me know if there's more, thank you!

**Comment by AllenFang** [comment]
@NixonK thanks your contribution 👍

**Comment by AllenFang** [comment]
@NixonK BTW, could you please change to the `develop` branch?

**Comment by NixonK** [comment]
@AllenFang Ah is the branch correct now? It is now rebased on top of origin/develop.

**Pull request closed without merging**